run-all.sh

```diff
@@ -19,8 +19,9 @@
 # Fail if anything fails.
 set -euv
 
-./run-test.sh libraryBuilds/com.google.code.gson-gson
-./run-test.sh libraryBuilds/org.joda-joda-convert
-./run-test.sh libraryBuilds/joda-time-joda-time
-./run-test.sh libraryBuilds/org.joda-joda-primitives
-./run-test.sh libraryBuilds/org.apache.commons-commons-lang3
+./run-test.sh com.google.code.gson-gson
+./run-test.sh org.joda-joda-convert
+./run-test.sh joda-time-joda-time
+./run-test.sh org.joda-joda-primitives
+./run-test.sh org.apache.commons-commons-lang3
+./run-test.sh com.fasterxml.jackson.core
```

Comment:
I'm not sure why the CI passed, given that I changed run-all.sh to use the full path (so autocomplete on the command line would work). Please check that out by comparing this to the repo version:

https://github.com/j2objc-contrib/j2objc-common-libs-e2e-test/blob/master/run-all.sh